core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/workflow/AbstractWorkflow.java

```diff
@@ -0,0 +1,44 @@
+package uk.ac.bbsrc.tgac.miso.core.data.workflow;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public abstract class AbstractWorkflow implements Workflow {
+  private Progress progress;
+
+  @Override
+  final public Progress getProgress() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)